seed/static/seed/js/controllers/mapping_controller.js

```diff
@@ -567,9 +567,31 @@ angular.module('BE.seed.controller.mapping', [])
         });
       };
 
+      var display_cached_column_mappings = function () {
+        var cached_mappings = JSON.parse($scope.import_file.cached_mapped_columns);
+        _.forEach($scope.mappings, function (col) {
+          var cached_col = _.find(cached_mappings, {from_field: col.name, to_table_name: col.suggestion_table_name})
+          col.suggestion_column_name = cached_col.to_field;
+          col.from_units = cached_col.from_units;
+
+          // If available, use display_name, else use raw field name.
+          var mappable_column = _.find(
+            $scope.mappable_property_columns.concat($scope.mappable_taxlot_columns),
+            {column_name: cached_col.to_field, table_name: cached_col.to_table_name}
+          )
+          if (mappable_column) {
+            col.suggestion = mappable_column.display_name;
+          } else {
+            col.suggestion = cached_col.to_field;
+          }
```

Comment:
The only time the `else` case would be used is if a Column was deleted. As I understand it, this should never never happen as users don't have the ability to delete Columns, but the fail safe was put in place just in case. Let me know if it actually is possible for users to delete Columns for an org.

Reply:
Yes, at the moment, users can't delete columns. So I think this is okay.